bots/jabber_mirror_backend.py

```diff
@@ -90,15 +96,18 @@ def __init__(self, jid, password, rooms):
         self.register_plugin('xep_0199') # XMPP Ping
 
     def set_zulip_client(self, client):
+        # type: (zulip.Client) -> None
         self.zulip = client
 
     def session_start(self, event):
+        # type: (Any) -> None
```

Comment:
It might be another task to do remember that here its not such easy to remove it as in JS. You need to modify all executions of the function